android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -768,6 +769,11 @@ protected void onSaveInstanceState(Bundle outState)
         fragment.saveAltitudeChartState(outState);
     }
 
+    if(mNavigationController != null)
```

Comment:
Very good. Please select this code  and format it, **only code which written by you** not whole file, because the white space between work 'if' and '(' is still missed. Fix it please.